nextflow.config

```diff
@@ -393,7 +395,6 @@ profiles {
             projectDir = '/work/project'
             pod = [
                 [label: 'pod-type', value: 'nextflow-worker'],
-                [env: 'NXF_DEBUG', value: '3'],
```

Comment:
Removing as the debug logs doesn't provide much value. Will add it back as needed.